File: scratch/remove_emojis.py
import os

directory = 'lib/l10n'
for filename in os.listdir(directory):
    if filename.endswith('.arb'):
        filepath = os.path.join(directory, filename)
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        if ' 😍' in content:
            content = content.replace(' 😍', '')
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f'Updated {filepath}')

File: scratch/remove_emojis.dart
import 'package:streambeats/l10n/app_localizations.dart';
import 'dart:io';

void main() {
  final dir = Directory('lib/l10n');
  final files = dir.listSync().whereType<File>().where((f) => f.path.endsWith('.arb'));
  for (final file in files) {
    String content = file.readAsStringSync();
    if (content.contains(' 😍')) {
      content = content.replaceAll(' 😍', '');
      file.writeAsStringSync(content);
      print('Updated ${file.path}');
    }
  }
}
